Improve the way strings are handled when writing to files (#118)
demisto/demisto-py: CHANGELOG.md

[PyPI History][1]
[1]: https://pypi.org/project/demisto-py/#history

## 3.2.12

> * Fixed an issue where download command failed do to str/bytes types.

Reviewer: ```suggestion
* Fixed an issue when the download command failed to convert bytes types to string.
```
